src/main/java/com/appian/intellij/k/actions/KServerProcessHandler.java

```diff
@@ -90,6 +92,58 @@ void execute(ConsoleView console, String q) {
       // close connection to force reconnection on any error other than
       // the error from q instance
       closeConnection();
+    } finally {
+      if (console instanceof ConsoleViewImpl) {
+        ((ConsoleViewImpl)console).requestScrollingToEnd();
```

Comment:
@YuriyYatsyk added this to make sure console scrolls to the end when executing actions. Otherwise, it's hard to know if something happened (unless the user manually scrolls).